saleor/page/models.py

```diff
@@ -0,0 +1,47 @@
+import datetime
+
+from django.core.validators import validate_slug
+from django.db import models
+from django.db.models import Q
+from django.urls import reverse
+
+
+class PageManager(models.QuerySet):
+    def public(self):
+        today = datetime.date.today()
+        return self.filter(
+            Q(visible=True),
+            Q(available_on__lte=today) | Q(available_on__isnull=True))
+
+    def get_available(self, allow_draft=False):
+        if not allow_draft:
+            return self.public()
+        return self
+
+
+class Page(models.Model):
+    url = models.CharField(
+        max_length=100, db_index=True, validators=[validate_slug])
+    title = models.CharField(max_length=200)
+    content = models.TextField()
+    created = models.DateTimeField(auto_now_add=True)
+    visible = models.BooleanField(default=False)
```

Comment:
It is a good convention to name boolean field with the prefix `is_`, so we could rename it to `is_visible`.